mezzanine/core/templatetags/mezzanine_tags.py

```diff
@@ -559,14 +559,13 @@ def admin_app_list(request):
                 DeprecationWarning)
         in_menu = in_menu and model_admin.has_module_permission(request)
         if in_menu and request.user.has_module_perms(opts.app_label):
-            perms = model_admin.get_model_perms(request)
             admin_url_name = ""
-            if perms["change"]:
+            if model_admin.has_change_permission(request, model):
```

Comment:
@ryneeverett I have django-filer integrated and this is the only app, which really throws an error here.